cluster/src/main/java/com/hartwig/pipeline/calling/structural/gridss/Gridss.java

```diff
@@ -103,7 +108,13 @@ private static String basename(final String filename) {
 
     @Override
     public VirtualMachineJobDefinition vmDefinition(final BashStartupScript bash, final ResultsDirectory resultsDirectory) {
-        return VirtualMachineJobDefinition.structuralCalling(bash, resultsDirectory);
+        return ImmutableVirtualMachineJobDefinition.builder()
+                .imageFamily(IMAGE_FAMILY)
```

Comment:
I think this breaks command line image overrides.

Reply:
I am afraid I don't fully understand what you mean, could you elaborate?